packages/transition-frontend/src/services/routing/RoutingUtils.ts

```diff
@@ -16,7 +16,7 @@ import { TransitAccessibilityMapWithPolygonResult } from 'transition-common/lib/
 export const calculateRouting = async (
     routing: TransitRouting,
     updatePreferences = false,
-    options: { isCancelled?: () => boolean; [key: string]: any } = {}
+    options: { isCancelled?: () => boolean; removeWalkOnlyPath?: boolean; [key: string]: any } = {}
```

Comment:
I the long term, I think we should reverse the logic of the field and have a "addWalkOnlyPath" option, when we would want to have it and remove it by default. Could be a TODO+Issue, but something to keep in mind